hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -11,7 +11,7 @@
 from threatexchange.signal_type.pdq_index import PDQIndex
 
 from hmalib import metrics
-from hmalib.models import PDQMatchRecord, Label
+from hmalib.models import PDQMatchRecord, Label, MatchMessage, DatasetMatchDetails
```

Comment:
Wow! How was the matcher working prior to this? 

Makes me wonder. What's `mypy` doing?

Reply:
because nothing depended on MatchMessage yet and we write to the dydb before trying to use the class

good question...